src/core/auth/atoms/userWasLanded.ts

```diff
@@ -0,0 +1,27 @@
+import config from '~core/app_config';
+import { createAtom } from '~utils/atoms';
+import { userResourceAtom } from './userResource';
+
+export const userWasLandedAtom = createAtom(
+  {
+    userResourceAtom,
+  },
+  ({ get }, state: 'yes' | 'no' | 'pending' = 'pending') => {
+    const userModel = get('userResourceAtom');
+    const userLoaded = userModel.data && !userModel.loading;
+    if (userLoaded) {
```

Comment:
This specific landing redirect functionality is feature, not core thing, and I [planned to move in into Aboutmode feature](https://kontur.fibery.io/Tasks/Task/Refactor-About-mode-to-perform-redirect-inside-feature,-not-in-Routes-(decoupling)-13264) - it should be possible to disable it via feature flag.

Reply:
agree